testsuite/features/step_definitions/api_common.rb

```diff
@@ -479,7 +479,7 @@ def wait_action_complete(actionid, timeout: DEFAULT_TIMEOUT)
   channel = if arch != 'x86_64'
               'test-channel-i586'
```

Comment:
missing `fake-` probably

Reply:
Apparently did not update this name yet, good catch